shared/extension/Script.hpp

```diff
@@ -375,10 +375,10 @@ namespace YAML {
                 rhs = {node["id"].as<std::string>(),
                        node["position"].as<float>(),
                        node["gain"].as<float>(),
-                       node["torque"] != nullptr ? node["torque"].as<float>() : 100};
+                       !node["torque"].IsNull() ? node["torque"].as<float>() : 100};
```

Comment:
I believe it should be this. `null` is not the same as not existing, it's when you have null in the yaml file
My guess is clang-tidy noticed that it should be `nullptr` and made that check but now it's changed
```suggestion
                       !node["torque"] ? node["torque"].as<float>() : 100};
```